### ISSUE TITLE
Tab button no longer has aria-selected="false" 

### ISSUE THREAD

**Issue by sambigelow** [chakra-ui/chakra-ui#2982]
# 🐛 Bug report

I think this is a regression from v0. The tabButton used to be calculated [like this](https://github.com/chakra-ui/chakra-ui/blob/v0/packages/chakra-ui/src/Tabs/index.js#L32) but in v1 Chakra is using `ariaAttr` which returns undefined if it is not `true`. However, as far as I can tell, this is supposed to be true or false and always exist on the tab buttons

[source1](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role)
[source2](https://www.maxability.co.in/2016/03/08/aria-selected-state/)

## 💥 Steps to reproduce

You can view `aria-selected="false"` missing in the [v1 examples](https://chakra-ui.com/docs/components/tabs) and you can view it existing in the [v0 examples](https://v0.chakra-ui.com/tabs)

## 💻 Link to reproduction

I think it's easy enough to see in the examples on the website.

## 🧐 Expected behavior

`aria-selected` should always be present on the `Tab` button

## 🧭 Possible Solution

I am not quite sure why we want `ariaAttr` to be undefined in most cases, but that seemed purposeful. Maybe just change [this line](https://github.com/chakra-ui/chakra-ui/blob/develop/packages/tabs/src/use-tabs.ts#L341) to `"aria-selected": Boolean(ariaAttr(isSelected))` but that is to be tested.

## 🌍 System information

| Software         | Version(s) |
| ---------------- | ---------- |
| Chakra UI        | 1.0.3 |
| Browser          | Firefox |
| Operating System | OSX |

**Comment by with-heart** [comment]
Additional context:

- Relevant `wai-aria` spec: https://www.w3.org/TR/wai-aria-1.0/states_and_properties
- [6.2.4](https://www.w3.org/TR/wai-aria-1.0/states_and_properties#propcharacteristic_value): spec for the possible values for an attribute.
- [`aria-selected` (state)](https://www.w3.org/TR/wai-aria-1.0/states_and_properties#aria-selected): spec for `aria-selected`. It's fairly difficult to parse.
- [ARIA: tab role (MDN)](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role): explains the `tab` role, and seems to make it pretty clear that elements with `role="tab"` should have `aria-selected="true"` or `aria-selected="false"`.

**Comment by segunadebayo** [comment]
I just pushed a fix for this. Thanks for catching it @sambigelow
